test_runner/regress/test_physical_and_logical_replicaiton.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+import time
+
+from fixtures.neon_fixtures import NeonEnv, logical_replication_sync
+
+
+def test_physical_and_logical_replication(neon_simple_env: NeonEnv, vanilla_pg):
+    env = neon_simple_env
+
+    n_records = 1000000
+
+    primary = env.endpoints.create_start(
+        branch_name="main",
+        endpoint_id="primary",
+        config_lines=["min_wal_size=32MB", "max_wal_size=64MB"],
+    )
+    p_con = primary.connect()
+    p_cur = p_con.cursor()
+    p_cur.execute("CREATE TABLE t(pk bigint primary key, payload text default repeat('?',200))")
+    p_cur.execute("create publication pub1 for table t")
+
+    # start subscriber
+    vanilla_pg.start()
+    vanilla_pg.safe_psql("CREATE TABLE t(pk bigint primary key, payload text)")
+    connstr = primary.connstr().replace("'", "''")
+    vanilla_pg.safe_psql(f"create subscription sub1 connection '{connstr}' publication pub1")
+
+    time.sleep(1)
+    secondary = env.endpoints.new_replica_start(
+        origin=primary,
+        endpoint_id="secondary",
+        config_lines=["min_wal_size=32MB", "max_wal_size=64MB"],
+    )
+    s_con = secondary.connect()
+    s_cur = s_con.cursor()
+
+    for pk in range(n_records):
+        p_cur.execute("insert into t (pk) values (%s)", (pk,))
+
+    # Wait at least 15 seconds until LR snapshots are persisted
+    time.sleep(16)
```

Comment:
Where does the 15 seconds come from?

Reply:
15 seconds is period of dumping LR snapshots.
Actually I am very sorry, but I was not abler to reproduce the reported problem (`Attempt to persist LR stuff at replica cause cannot make new WAL entries during recovery`) with this test. Yes, LR slots are "inherited" by replica. But no subscription can be activated at replica and so no LR workers are started. As a result this slots are inactive and no LR snapshots are saved. 

I still trying to reproduce it...

Reply:
I have updated the test.
Now it reproduces the problem.